addons/udes_stock_cron/models/stock_picking.py

```diff
@@ -265,9 +268,47 @@ def _process_pickings(self):
         if unsatisfied:
             # Unreserve if the picking type cannot handle partials or it
             # can but there is nothing allocated (no stock.move.lines)
-            if not self.picking_type_id.u_handle_partials or not mls:
+            # If there are several picking types, they should have all same handle_partials,
+            # can be improved to handle it better if necessary.
+            handle_partial = self.picking_type_id[0].u_handle_partials
```

Comment:
nit, should pobably enforce this, not sure if there is a better way
```python
handle_partial = set(picking_types.mapped("u_handle_partials"))
if len(handle_partial) > 1:
    raise ...
if handle_partial = {False} or not mls:
```

Reply:
We had a discussion about handle partials that should be all same, but i think you are right better safe than sorry